src/flexalloc_util.h

```diff
@@ -31,22 +32,27 @@
  * @brief Activates error handling if condition is non zero.
  *
  * @param condition zero means no error
- * @param message The message to use in case there is an error
  * @param f The file name where the error occured
  * @param l The line where the error occurred
- * @return condition is returned when errno is 0, otherwise errno is returned
+ * @param fmt The message to use in case there is an error
+ * @param ... vriadic args for fmt. Can be empty.
+ * @return condition is returned.
  */
 static inline int
-fla_err_fl(const int condition, const char * message, const char * f,
-           const int l )
+fla_err_fl(const int condition, const char * f, const int l, const char * fmt, ...)
 {
   if(condition && condition < 2001)
```

Comment:
I guess this refers to `FLA_ERR_ALL_SLABS_USED`

Is there a way to unify processing so that we are not branching on value ranges ?
Failing that, can we introduce a specific define and use it here instead of a magic (raw) value?